td4m3/src/main/java/ca/td/td4m3/mortgage/activity/MortgageActivity.kt

```diff
@@ -1,15 +1,29 @@
 package ca.td.td4m3.mortgage.activity
 
+import ca.td.td4m3.R
 import ca.td.td4m3.mortgage.adapter.MortgageActivityAdapter
 import ca.td.td4m3.mortgage.presenter.MortgageActivityPresenter
+import ca.td.td4m3.mortgage.view.MortgageActivityView
 import com.ngam.rvabstractions.AbstractActivity
 import com.ngam.rvabstractions.AbstractClassProperties
 
-class MortgageActivity: AbstractActivity<MortgageActivityPresenter, MortgageActivityAdapter>(){
+class MortgageActivity : AbstractActivity<MortgageActivityPresenter, MortgageActivityAdapter>(),
+        MortgageActivityView {
 
-    override fun setProperties(): AbstractClassProperties<MortgageActivityPresenter, MortgageActivityAdapter> {
-        presenter = MortgageActivityPresenter()
-        adapter = MortgageActivityAdapter()
+    override fun onResume() {
+        super.onResume()
+        val padding = resources.getDimensionPixelOffset(R.dimen.card_top_spacing)
```

Comment:
Why in onResume and not in onCreate?

Reply:
Null pointer exception for listview. onResume is perfect.

Reply:
Put in onCreate.